internal/handlers/tigris/msg_aggregate.go

```diff
@@ -103,7 +104,7 @@ func (h *Handler) MsgAggregate(ctx context.Context, msg *wire.OpMsg) (*wire.OpMs
 
 		var s aggregations.Stage
 		if s, err = aggregations.NewStage(d); err != nil {
-			return nil, err
+			return nil, lazyerrors.Error(err)
```

Comment:
I added more `lazyerror` errors to simplify debugging (to see when things fail when they fail). If you think some of these errors shouldn't be "lazy", please correct me.

Reply:
I'm not particularly accustomed to using much lazyerror but it's probably okay :hugs: